package.json

```diff
@@ -159,5 +159,8 @@
     "build:prod": "webpack --mode=production --define-process-env-node-env=production",
     "test": "jest",
     "watch": "webpack --watch"
+  },
+  "resolutions": {
```

Comment:
This change can be reverted with the above fix. I just tested it, there is no error even without the `resolution` block.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/498/commits/23fcc761bc85dcb7077673d1a288a7e603cec03f